bin/subscribeUsersToPersonalizedDigest.ts

```diff
@@ -1,45 +1,52 @@
 import '../src/config';
 import createOrGetConnection from '../src/db';
+import fs from 'fs/promises';
 
-(async (): Promise<void> => {
-  const createdFromArgument = process.argv[2];
-  const createdToArgument = process.argv[3];
+const parseCSV = <T>(csv: string, splitBy: string): T[] => {
```

Comment:
We also have import { parse } from 'csv-parse'; installed.
see for instance notificationSubscribedUsers

Reply:
I had that one from some other scripts of mine 😆 , did not wish to install dependencies, but also did not look if we have some already, yes I'll use the package. 

Manual CSV import don't think supports the timezone assignment, see https://github.com/dailydotdev/daily-api/pull/1466#issuecomment-1751183586

We could bottom line do CSV import and then run a query to set timezone based on the value of `user.timezone` column. 

We could also see if Dave can pull it and provide timezones in CSV. Again the above seemed fastest.

Reply:
Ah yeah the timezone was more around the initial CSV could potentially have it.
But also happy with your propsal tbh, don't want to spend to long on the task.